src/Uno.UI.Toolkit/MenuFlyoutExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.UI.Toolkit
+{
+#if __IOS__
+	[global::Foundation.PreserveAttribute(AllMembers = true)]
+#elif __ANDROID__
+	[Android.Runtime.PreserveAttribute(AllMembers = true)]
+#endif
+	public class MenuFlyoutExtensions
+	{
+		#region Property: OverrideIosCancelText
+		public static DependencyProperty OverrideIosCancelTextProperty { get; } = DependencyProperty.RegisterAttached(
+			"OverrideIosCancelText",
+			typeof(string),
+			typeof(MenuFlyoutExtensions),
+			new PropertyMetadata(default(string)));
+
+		public static string GetOverrideIosCancelText(MenuFlyout obj) => (string)obj.GetValue(OverrideIosCancelTextProperty);
+
+		public static void SetOverrideIosCancelText(MenuFlyout obj, string value) => obj.SetValue(OverrideIosCancelTextProperty, value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'obj' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5611164)